codingStandards/CodingStandard-Html.md

```diff
@@ -237,3 +237,34 @@
       </td>
     </tr>
   </table>
+
+- Always remember to include `rel="noopener noreferrer"` when using `target="_blank"` in links. 
+> Leaving this out would enable attackers to get hold of `window.opener` object, which, for example, can used to redirect to a malicious link. [\[Source\]](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.jlkn46vtk)
```

Comment:
- No need `for example`.
- Missing 'be' in `can used to`.
- Is it necessary to escape the square brackets?
- The anchor in the link should be cleaned up.